src/app/presentation/menu-shortcut/menu-shortcut.component.html

```diff
@@ -5,12 +5,13 @@
         <p><a href="/">Index</a></p>
       </li>
       <li class="menu-item" *ngFor="let route of routes">
-        <p><a [href]="route.path + '/0'">{{route.name}}</a></p>
+        <p><a [href]="route.path">{{route.name}}</a></p>
```

Comment:
can you test this? Open the thing, and check that once you go to the next slide URL is udpated. 

Also replace href with routerLink for #86